Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM calico/build:latest
+MAINTAINER projectcalico
+
+ADD . /code
+WORKDIR /code
+
+ARG K8S_VERSION=local
+ENV K8S_VERSION ${K8S_VERSION}
+
+RUN pip install pykube && pyinstaller calico.py -ayF && \
```

Comment:
Will do. Not sure I got it to adding the binaries. Nothing to build when? What about libs shown by ldd for the binaries? Shall I put them in the container?